pkgs/development/tools/analysis/smatch/default.nix

```diff
@@ -8,19 +8,36 @@
 , buildc2xml ? false
 , libllvm
 , libxml2
+, substituteAll
+, llvmPackages
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "smatch";
   version = "1.73";
 
   src = fetchFromGitHub {
     owner = "error27";
     repo = "smatch";
-    rev = version;
-    sha256 = "sha256-Pv3bd2cjnQKnhH7TrkYWfDEeaq6u/q/iK1ZErzn6bME=";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-Pv3bd2cjnQKnhH7TrkYWfDEeaq6u/q/iK1ZErzn6bME=";
   };
 
+  patches = [
+    (let
+      clang-major = lib.versions.major (lib.getVersion llvmPackages.clang-unwrapped);
+      clang-lib = lib.getLib llvmPackages.clang-unwrapped;
+
+      clang = "${clang-lib}/lib/clang/${clang-major}/include";
+      libc = "${lib.getDev stdenv.cc.libc}/include";
+    in substituteAll {
+      src = ./fix_include_path.patch;
+
+      inherit clang libc;
+    })
+  ];
+
+  enableParallelBuilding = true;
```

Comment:
I believe that parallel building is enabled by default.

Reply:
I is `false` by default, *"some build systems with good support for parallel building including cmake, meson, and qmake will set it to true"*
> https://nixos.org/manual/nixpkgs/stable/#var-stdenv-enableParallelBuilding


without: `56.595s`, with: `9.964s`